src/Symfony/Component/HttpKernel/Tests/EventListener/RouterListenerTest.php

```diff
@@ -155,4 +155,19 @@ public function getLoggingParameterData()
             array(array(), 'Matched route "n/a".'),
         );
     }
+
+    /**
+     * @expectedException \Symfony\Component\HttpKernel\Exception\BadRequestHttpException
+     */
+    public function testSubRequestWithBadHost()
```

Comment:
It's not about sub request. But master request

Reply:
"Sub" removed, thanks

Reply:
oups, it's already merged :) PR welcome then :P